schemas/json/layout/layout.schema.v1.json

```diff
@@ -0,0 +1,213 @@
+{
```

Comment:
Add `$id´. Like this: https://github.com/Altinn/altinn-cdn/blob/master/schemas/json/policy/policy.schema.v1.json#L2

https://json-schema.org/understanding-json-schema/basics.html#declaring-a-unique-identifier